### PR TITLE
 Tab UI sample - Welcome Tab fixes

### PR DESCRIPTION
**Pull request by Jegadeesh-MSFT**

### CODE REVIEW

**Review by Pawank-MSFT — APPROVED** [review]
Looks correct, Approving!

**Comment by Pawank-MSFT** [comment]
Executed Build pipeline:
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/7899922522

**Review by Harikrishnan-MSFT — CHANGES_REQUESTED** [review]

File: samples/tab-ui-templates/ts/src/app/static_tabs/welcome.tsx [hunk 1]
```diff
@@ -21,38 +21,42 @@ export function WelcomeTab() {
...
-      <Text
-        style={{ marginBottom: "3rem", opacity: ".65", maxWidth: "50rem" }}
-      >
-        The Teams Sample App can help you better understand how your app should
-        look and behave in Teams depending on the scenario. Select a UI template
-        above (for example, Forms) to get started. To learn more, see the
-        following resources.
-      </Text>
+        <Text
+          style={{ marginBottom: "3vh", opacity: ".65", maxWidth: "50rem" }}
```
[the remark this answers is not in the document]
Comment (Jegadeesh-MSFT) [reply samples/tab-ui-templates/ts/src/app/static_tabs/welcome.tsx:25]: will be Fixed in future PR.

File: samples/tab-ui-templates/ts/src/app/static_tabs/welcome.tsx [hunk 2]
```diff
@@ -21,38 +21,42 @@ export function WelcomeTab() {
...
+          look and behave in Teams depending on the scenario. Select a UI template
+          above (for example, Forms) to get started. To learn more, see the
+          following resources.
+        </Text>
       </div>
+      <br /><br />
       <Text size={500} weight="bold" as="h2">
         Resources
       </Text>
+      <style>
```
[the remark this answers is not in the document]
Comment (Jegadeesh-MSFT) [reply samples/tab-ui-templates/ts/src/app/static_tabs/welcome.tsx:37]: will be Fixed in future PR.

File: samples/tab-ui-templates/ts/src/app/static_tabs/welcome.tsx [hunk 3]
```diff
@@ -21,38 +21,42 @@ export function WelcomeTab() {
...
+        {`
+          .ui-grid {
+            display: inline-flex;
+            margin-top: 30px;
+          }
+        `}
+      </style>
       <Grid
         styles={{
-          gridGap: "2rem",
```
[the remark this answers is not in the document]
Comment (Jegadeesh-MSFT) [reply samples/tab-ui-templates/ts/src/app/static_tabs/welcome.tsx:38]: will be Fixed in future PR.

**Pull request merged**